changelog_entries/change_filtering_filter_weapon.md

```diff
@@ -0,0 +1,2 @@
+ ### WML Engine
+   * filter "damage" and filter "number" in [filter_attack/weapon] matche with real damage or attack number values except respectively in [damage] and [attacks]
```

Comment:
IIUC, this was the first implementation, but not the current state of the PR.

I think it should be "Add weapon filter keys `effective_damage` and `effective_number`, which account for time of day, leadership, being slowed, etc."

Reply:
except what effective_damage don't account time of day or leadership because modified_damage calculation was before [leadership] or Tod, and slow, imut rename filter like special_damage or abandon this idea(this implementation is for integrate [damage] or [attacks] but that all)

Reply:
Ah, I misunderstood what it would be used for then.